storage/simple_tree.hpp

```diff
@@ -60,12 +61,12 @@ class SimpleTree
     return Value() < other.Value();
   }
 
-  /// sorts siblings independently on each level by default
-  void Sort(bool onlySiblings = false)
+  /// sorts children independently on each level by default
+  void Sort(bool onlyChildren = false)
```

Comment:
It's unclear for me why do we need onlyChildren flag at all. The comment claims that this function sorts children, so what the purpose of the flag?